### ISSUE TITLE
Don't know how to update profile pic or profile banner

### ISSUE THREAD

**Issue by Ternoc** [abraham/twitteroauth#361]
Hello,
I need help because I can't find a way to update profile pic or banner pic using the class. 
I tried with

``` php
$media1 = $connection->upload('media/upload', array('media' => '/pic.jpg'));
$parameters = array(
    'image' => implode(',', array($media1->media_id_string)),
);
$result = $connection->post('account/update_profile_image', $parameters);
```

but it return [error] => media type unrecognized.

If someone can help me :)

**Comment by abraham** [comment]
The `media/upload` endpoint can't be used for profile image uploads and profile image uploads have not been implemented yet.

**Comment by 2s2d** [comment]
very bad, we will wait for the implementation of

**Comment by zntfdr** [comment]
🌈 +1 for this issue 🌈
I use this library with [several bots](https://twitter.com/zntfdr/lists/zntfdr-s-bots/members) and I'd love to have a dynamic profile picture 😊

**Comment by abraham** [comment]
This should get fixed with #685

**Comment by jaehaerys-x** [comment]
Hi sir @abraham, I see that this was fixed with #685. Can you please help me how to implement it? I am still getting the same error: media type unrecognized. Thank you.

**Comment by jaehaerys-x** [comment]
> Hi sir @abraham, I see that this was fixed with #685. Can you please help me how to implement it? I am still getting the same error: media type unrecognized. Thank you.

Sorry please ignore my message. Got it solved already. It worked when I added base64_encode.

$parameters = [
		'image' => base64_encode(file_get_contents($filename))
		];
				
$response = $connection->post('account/update_profile_image', $parameters);
